packages/select/test/items.test.js

```diff
@@ -0,0 +1,76 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import './not-animated-styles.js';
+import '../vaadin-select.js';
+
+describe('items', () => {
+  let select, overlay, listBox;
+
+  beforeEach(() => {
+    select = fixtureSync(`<vaadin-select></vaadin-select>`);
+    select.items = [{ label: 'Option 1', value: 'value-1' }];
+    select.opened = true;
+    overlay = select._overlayElement;
+    listBox = overlay.querySelector('vaadin-list-box');
+  });
+
+  it('should render items', () => {
+    expect(listBox).to.be.ok;
```

Comment:
Worth mentioning that I tried to test the items API with snapshot tests, however, it turned out to be not so easy since the list-box is actually in the overlay that is outside the select in terms of the DOM structure. Decided to get by with unit tests for now.